app/controllers/discover.js

```diff
@@ -235,6 +298,15 @@ export default ApplicationController.extend({
 
         clearFilters() {
             this.set('facetFilters', Ember.Object.create());
+            let params = this.get('filterQueryParams');
+            for (var param in params) {
+                let key = params[param];
+                if (params.indexOf(key) > -1) {
+                    this.set(key, '');
```

Comment:
If the value is `null` then the URL looks like `?contributors=null&sources=null` which then breaks the page. I'll try and get them out of the url though.